src/modeling.py

```diff
@@ -1,3 +1,43 @@
+1r6PoYic
+import pandas as pd
+from sklearn.model_selection import train_test_split
+
+def train_test_split_df(df, target_col, test_percent, random_state):
+    """
+    Split a DataFrame into training and testing sets for machine learning.
+
+    Parameters:
+        df (pandas.DataFrame): The input DataFrame.
+        target_col (str): The name of the target column in the DataFrame.
+        test_percent (float): The percentage of data to use for testing (between 0 and 1).
+        random_state (int): The random state to ensure reproducibility.
+
+    Returns:
+        X_train (pandas.DataFrame): The training set features.
+        X_test (pandas.DataFrame): The testing set features.
+        y_train (pandas.Series): The training set target variable.
+        y_test (pandas.Series): The testing set target variable.
+
+    Prints:
+        Shape of each set to confirm dimension.
+    """
+    # Separate features (X) and target variable (y)
```

Comment:
Falta el try-except

Reply:
oh el try except. thank you. any suggestions como ponerlo?

Reply:
Eres la desarrolladora de la función, eso tienes que saberlo tú.

Reply:
si claro. sorry estoy un poco tonta. thanks